examples/configs/test_example.yml

```diff
@@ -0,0 +1,73 @@
+# lightning.pytorch==2.0.4
+seed_everything: true
+trainer:
+  accelerator: auto
+  strategy: auto
+  devices: auto
+  num_nodes: 1
+  precision: 32-true
+  callbacks: null
+  fast_dev_run: false
+  max_epochs: null
+  min_epochs: null
+  max_steps: -1
+  min_steps: null
+  max_time: null
+  limit_train_batches: null
+  limit_val_batches: null
+  limit_test_batches: null
+  limit_predict_batches: null
+  overfit_batches: 0.0
+  val_check_interval: null
+  check_val_every_n_epoch: 1
+  num_sanity_val_steps: null
+  log_every_n_steps: null
+  enable_checkpointing: null
+  enable_progress_bar: null
+  enable_model_summary: null
+  accumulate_grad_batches: 1
+  gradient_clip_val: null
+  gradient_clip_algorithm: null
+  deterministic: null
+  benchmark: null
+  inference_mode: true
+  use_distributed_sampler: true
+  profiler: null
+  detect_anomaly: false
+  barebones: false
+  plugins: null
+  sync_batchnorm: false
+  reload_dataloaders_every_n_epochs: 0
+  default_root_dir: null
+model:
+  architecture: null
+  model_config: {}
+  loss_function: null
+  lr: 0.001
```

Comment:
At least from the examples at the DL@MBL course 1e-4 worked better and I think that is the default in most networks, but if this typically works better in our networks then great.

Reply:
The best LR is highly data and model dependent, and there is not a good default that works well for everything. This LR works for 2.5D UNet with 64 patches per batch. For 2.5D LUNeXt the gradient will explode after a few epochs so I've been using 2e-4.